dev/ledger.md

```diff
@@ -820,13 +820,18 @@ and contains the following fields:
     of the entries follows the execution order of the `log`
     invocations. The maximum total number of `log` calls is 32, and the
     total size of all logged bytes is limited to 1024.
-  - Zero or more `InnerTxns`, encoded in an array `itx`. Each element of
-    `itx` records a successful invocation of the `tx_submit` opcode. Each
-    element will contain the transaction fields, encoded under `txn`, in
-    the same way that the top-level transaction is encoded, recursively,
-    including `ApplyData` that applies to the inner transaction.  Up to 16
-    `InnerTxns` may be present.
-    - InnerTxns are limited to `pay`, `axfer`, `acfg`, and `afrz` transactions.
+  - Zero or more `InnerTxns`, encoded in an array `itx`. Each element
```

Comment:
Are the innerTxns delimited in groups? Is there a difference in `EvalDelta` between two separate inner txns and a single 2-txn group?

Reply:
In order to avoid making more EvalDelta changes, the inner transactions of a single transactions are stored in a single linear list.  But groups can be distinguished by their identical group IDs. (Or, if there is no group ID, the inner transactions is a singleton.). Which, of course, it how groups are distinguished in blocks too, so it seems ok.